SWLOR.Game.Server/Feature/LootTableDefinition/SpaceLootTableDefinition.cs

```diff
@@ -20,20 +20,17 @@ public class SpaceLootTableDefinition : ILootTableDefinition
 
         private void Tier1()
         {
-            _builder.Create("SPACE_SYBIL")
-                .AddItem("elec_ruined", 20)
-                .AddItem("ref_tilarium", 3);
+            _builder.Create("SPACE_BASIC_T1")
```

Comment:
Please review these loot tables for anything that needs to be marked as a rare item. Right now none of them are, so Treasure Hunter won't work.